docs/code_gen_options.rst

```diff
@@ -149,21 +148,49 @@ as a function argument, for instance, the length is unknown.
 Debugging Options
 -----------------
 
-It may be desirable to have access to debug information regarding the contract execution. 
-However, this will lead to more instructions as well as higher gas usage. Hence, it is disabled by default,
-but can be enabled using CLI flags. Just make sure not to use them in production deployments.
+It is desirable to have access to debug information regarding the contract execution in the testing phase.
+Hence, debugging options are enabled by default, but can be disbaled using CLI flags. 
+Therefore, by default, debugging options are enabled; however, they can be deactivated by utilizing the command-line interface (CLI) flags.
+It is imperative to disable debugging options in release builds, as debugging would result in increased instructions and greater gas consumption.
+Solang provides three debugging options, namely debug prints, logging API return codes, and logging runtime errors.
```

Comment:
I think this sentence should be moved to the beginning of this chapter (`Debugging Options`). It should also say that these 3 options are exactly the ones being disable by the `--release` flag.